src/20240122_AaveV3Ethereum_RegisterADIScrollAdapter/AaveV3Ethereum_RegisterADIScrollAdapter_20240122.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {ChainIds} from 'aave-helpers/ChainIds.sol';
+
+interface ICrossChainForwarder {
```

Comment:
I would import the interface from [address-book](https://github.com/bgd-labs/aave-address-book/blob/a6cd3509403bb3c96afb4c55c7214f1257aa783f/src/common/ICrossChainController.sol#L769) as it would be much cleaner.

Reply:
Agree, better to keep the payload clean